templates/repo/actions/list.tmpl

```diff
@@ -19,6 +19,50 @@
 				</div>
 			</div>
 			<div class="twelve wide column content">
+				<div class="ui secondary filter stackable menu gt-je">
+					<!-- Actor -->
+					<div class="ui {{if not .Actors}}disabled{{end}} dropdown jump item">
+						<span class="text">
+							{{.locale.Tr "actions.runs.actor"}}
+							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						</span>
```

Comment:
It is a wrong layout  https://fomantic-ui.com/modules/dropdown.html#dropdown

The standard layout should be `.ui.dropdown > .dropdown.icon`

Reply:
Updated in https://github.com/go-gitea/gitea/pull/25118/commits/738df8f18f8515a2c3c82483300e4570d9307e57